pkgs/by-name/bo/bombsquad/package.nix

```
@@ -0,0 +1,96 @@
+{
+  stdenv,
+  lib,
+  python3,
+  SDL2,
+  mesa,
+  libGL,
+  openal,
+  libvorbis,
+  fetchurl,
+  makeWrapper,
+  patchelf,
+  copyDesktopItems,
+  makeDesktopItem
+}:
+let
+  runtimeDependencies = [
+    python3
+    SDL2
+    libvorbis
+    mesa
+    stdenv.cc.cc.lib
+    libGL
+    openal
+  ];
+in
+stdenv.mkDerivation (finalAttrs: {
+  version = "1.7.33";
+  pname = "bombsquad";
+
+  src = fetchurl {
+    sha256 = "sha256-pfimBt1s4QP+6buPvTkSdmt5zj2NpXqJUfbxRirfz/o=";
+    url = "http://files.ballistica.net/bombsquad/builds/BombSquad_Linux_x86_64_${finalAttrs.version}.tar.gz";
+  };
+
+  sourceRoot = "BombSquad_Linux_x86_64_${finalAttrs.version}";
+  dontConfigure = true;
+  dontBuild = true;
+  nativeBuildInputs = [
+    patchelf
+    makeWrapper
+    copyDesktopItems
+  ];
+
+  desktopItems = [
+    (makeDesktopItem rec {
+      name = "Bombsquad";
+      exec = finalAttrs.pname;
+      icon = fetchurl {
+        url = "https://www.froemling.net/wp-content/uploads/2018/02/icon_clipped-1-150x150.png";
+        sha256 = "sha256-x08rekyPzHe2FdQzo1EIZ3Rgg/OAcTbWCHoiKuGRBvU=";
+      };
+      comment = finalAttrs.meta.description;
+      desktopName = name;
+      genericName = name;
+      startupWMClass = ".bombsquad-wrapped";
+      categories = [ "Game" ];
+    })
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    copyDesktopItems
+    mkdir -p $out/bin
+    cp -r ba_data $out/ba_data
+    cp ${finalAttrs.pname} $out/bin/${finalAttrs.pname}
+    runHook postInstall
+  '';
+
+  postFixup = ''
+    patchelf \
+      --set-rpath ${ lib.makeLibraryPath runtimeDependencies } \
+      --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+      $out/bin/${finalAttrs.pname}
+    wrapProgram $out/bin/${finalAttrs.pname} --chdir "$out"
+  '';
+
+  meta = with lib; {
+    homepage = "https://ballistica.net/";
+    description = "A party game to blow up your friends in various minigames from CTF to hockey.";
+    longDescription = ''
+      Blow up your friends in mini-games ranging from capture-the-flag to hockey!
+      Featuring 8 player local/networked multiplayer, gratuitous explosions, advanced ragdoll face-plant physics,
+      pirates, ninjas, barbarians, insane chefs, and more.
+
+      BombSquad supports touch screens as well as a variety of controllers so all your friends can get in on the action.
+      You can even use phones and tablets as controllers via the free 'BombSquad Remote' app.
+
+      Bombs away!
+    '';
+    platforms = [ "x86_64-linux" ];
+    sourceProvenance = [ sourceTypes.binaryNativeCode ];
+    maintainers = with maintainers; [ coffeeispower ];
+    mainProgram = finalAttrs.pname;
```

```suggestion
    mainProgram = "bombsquad";
```

https://github.com/NixOS/nixpkgs/issues/277994